Raspi/Test/mock_wireless_module.py

```diff
@@ -5,24 +5,24 @@
 from MockSensor import MockSensor
 
 parser = argparse.ArgumentParser(
-    description='MQTT wireless sensor test script that sends fake data',
+    description='MQTT wireless module test script that sends fake data',
     add_help=True)
 parser.add_argument(
     '-t', '--time', action='store', type=int,
     default=float('Inf'), help="""Length of time to send data in seconds
     (duration). If nothing is selected it will continuously send out data.""")
 parser.add_argument(
-    '-r', '--rate', action='store', type=float, default=1,
-    help="""Rate of data in data sent per second. Default is 1 data pack sent
+    '-r', '--rate', action='store', type=float, default=5,
```

Comment:
Any reason you set the default to 5?

Reply:
Because on this slack thread I learned the rate was currently 9hz but was going to be changed to 5hz so I changed it. Thought it represented the read data more closely.

https://monashhumanpower.slack.com/archives/CCD8JK739/p1579558297007000

Reply:
If the data rate changes its pretty easy to change back.

Reply:
Ohhhhh, good initiative! But I think we should leave it at 1Hz for now as it's a lot easier to keep track (when testing). If we do end up using 5Hz all the time, then we'll change the default. Sorry for the confusion!